src/transformers/commands/user.py

```diff
@@ -173,6 +207,85 @@ def run(self):
         print("Done")
 
 
+class ListReposObjsCommand(BaseUserCommand):
+    def run(self):
+        token = HfFolder.get_token()
+        if token is None:
+            print("Not logged in")
+            exit(1)
+        try:
+            objs = self._api.list_repos_objs(token, organization=self.args.organization)
+        except HTTPError as e:
+            print(e)
+            print(ANSI.red(e.response.text))
+            exit(1)
+        if len(objs) == 0:
+            print("No shared file yet")
+            exit()
+        rows = [[obj.filename, obj.lastModified, obj.commit, obj.size] for obj in objs]
+        print(tabulate(rows, headers=["Filename", "LastModified", "Commit-Sha", "Size"]))
+
+
+class RepoCreateCommand(BaseUserCommand):
+    def run(self):
+        token = HfFolder.get_token()
+        if token is None:
+            print("Not logged in")
+            exit(1)
+        try:
+            stdout = subprocess.run(["git", "--version"], capture_output=True).stdout.decode("utf-8")
```

Comment:
I'm currently getting a:

```
TypeError: __init__() got an unexpected keyword argument 'capture_output'
```

thrown here. I'll start investigating it now!

Reply:
Related to: https://stackoverflow.com/q/53209127

Reply:
The simplest most supported way is `output = subprocess.check_output(["git", "--version"])`

Reply:
Good catch, will make sure it runs on PY3.6

Reply:
@stas00 @julien-c Just as a reminder: default encoding on Windows is still `cp1252`. In this case, it probably will not matter because the output does not contain any weird characters. However, to be safe (and to be really cross-platform), you may want to opt for:

```python
import locale

output = subprocess.check_output(["git", "--version"]).decode(locale.getpreferredencoding())
```

Reply:
I previously added a comment about default encodings (which is cp1252 on Windows by default). However, there are many factors that determine the encoding for specific things (`locale`, sys encoding, environment variables but even OS and Python versions!). As an example: the default decoding differs for me when using my editors built-in terminal and PowerShell and CMD. On top of that `locale.getpreferredencoding` differs from `sys.stdin.encoding`.

I do not have the time to look into this further to figure out what the best, cross-platform, cross-Python-version way is to deal with this. As for now, since the output of `git --version` is simple enough without special chars, I think we can stick with UTF-8.